Description: Remove external assets from documentation.
 Upstream incldues a PayPal donation form in it's sphinx templates for which the
 submit button used an external image and which was replaced by a simple
 <button/> element. Also a gittip button is loaded inside an <iframe/> which is
 removed. The logo image was loaded from the project website and is replaced
 with the local image file.
Author: Michael Fladischer <FladischerMichael@fladi.at>
Last-Update: 2014-02-11
Forwarded: not-needed

--- a/docs/.templates/sidebarintro.html
+++ b/docs/.templates/sidebarintro.html
@@ -1,5 +1,5 @@
 <p class="logo"><a href="{{ pathto(master_doc) }}">
-  <img class="logo" src="http://cloud.github.com/downloads/celery/celery/celery_128.png" alt="Logo"/>
+  <img class="logo" src="{{ pathto("_images/celery_128.png", 1) }}" alt="Logo"/>
 </a></p>
 <div id="donate">
     <b>Please donate:</b>
@@ -7,9 +7,5 @@
         <input type="hidden" name="cmd" value="_s-xclick">
         <input type="hidden" name="hosted_button_id" value="BPCLP83DFF2N2">
     </form>
-    <img onclick="$('#paypal_donate').submit();"
-        src="http://celeryproject.org/static/img/icon-paypal.png" alt="amex">
-    <iframe style="border: 0; margin: 0; padding: 0;"
-            src="https://www.gittip.com/ask/widget.html"
-            width="48pt" height="22pt"> </iframe>
+    <button onclick="$('#paypal_donate').submit();" alt="PayPal">PayPal</button>
 </div>
--- a/docs/.templates/sidebarlogo.html
+++ b/docs/.templates/sidebarlogo.html
@@ -1,5 +1,5 @@
 <p class="logo"><a href="{{ pathto(master_doc) }}">
-  <img class="logo" src="http://cloud.github.com/downloads/celery/celery/celery_128.png" alt="Logo"/>
+  <img class="logo" src="{{ pathto("_images/celery_128.png", 1) }}" alt="Logo"/>
 </a></p>
 <div id="donate">
     <b>Please donate:</b>
@@ -7,9 +7,5 @@
         <input type="hidden" name="cmd" value="_s-xclick">
         <input type="hidden" name="hosted_button_id" value="BPCLP83DFF2N2">
     </form>
-    <img onclick="$('#paypal_donate').submit();"
-        src="http://celeryproject.org/static/img/icon-paypal.png" alt="amex">
-    <iframe style="border: 0; margin: 0; padding: 0;"
-            src="https://www.gittip.com/ask/widget.html"
-            width="48pt" height="22pt"> </iframe>
+    <button onclick="$('#paypal_donate').submit();" alt="PayPal">PayPal</button>
 </div>
